website (As a sanity check)/script.js

```diff
@@ -0,0 +1,17 @@
+
+var successful = true;
+let form = document.querySelector('form');
+let main = document.querySelector('.container')
+let btn = document.querySelector('.submit')
+
+function formSubmit(){
+    if (successful) {
+         console.log(form)
+         form.style.visibility = 'hidden'
+         main.innerHTML = '<div class="container-2"><p class="success">Form Submited</p></div>'
```

Comment:
What kind of freak are you, this is the most **Obtuse** line of code I've ever seen